spec/requests/list_controller_spec.rb

```diff
@@ -199,8 +199,6 @@
   end
 
   describe '#private_messages_group' do
-    fab!(:user) { Fabricate(:user) }
```

Comment:
So the `user` was not necessary? 🤔

Reply:
There is already a user fabricated in the parent scope.